react/week4/src/components/UsersSearch.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from "react";
+import UsersContext from './UserContext';
+import MainTitle from './Headers';
+import './UsersSearch.css'
+
```

Comment:
It would great if you could add useDebounce hook here https://usehooks.com/useDebounce/ for setting the query for the search